src/test/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TestArmorTask.java

```diff
@@ -64,55 +56,9 @@ void testSlimefunArmor() throws InterruptedException {
 
         player.getInventory().setHelmet(helmet.clone());
         player.getInventory().setChestplate(new ItemStack(Material.DIAMOND_CHESTPLATE));
-        new ArmorTask(false).run();
+        new SlimefunArmorTask().run();
 
         // Check if all Potion Effects were applied
         Assertions.assertTrue(player.getActivePotionEffects().containsAll(Arrays.asList(effects)));
     }
-
-    @ParameterizedTest
-    @DisplayName("Test Radiation and Hazmat Suits")
-    @MethodSource("cartesianBooleans")
-    void testRadiactivity(boolean hazmat, boolean radioactiveFire) throws InterruptedException {
-        Player player = server.addPlayer();
-        TestUtilities.awaitProfile(player);
-
-        // Setting the time to noon, to exclude the Solar Helmet check
-        player.getWorld().setTime(16000);
-
-        ItemGroup itemGroup = TestUtilities.getItemGroup(plugin, "hazmat_suit_test");
-        SlimefunItemStack item = new SlimefunItemStack("MOCK_URANIUM_" + String.valueOf(hazmat).toUpperCase(Locale.ROOT) + "_" + String.valueOf(radioactiveFire).toUpperCase(Locale.ROOT), Material.EMERALD, "&aHi, I am deadly");
-        new RadioactiveItem(itemGroup, Radioactivity.VERY_DEADLY, item, RecipeType.NULL, new ItemStack[9]).register(plugin);
-
-        player.getInventory().setItemInMainHand(item.clone());
-        player.getInventory().setItemInOffHand(new ItemStack(Material.EMERALD_ORE));
-
-        if (hazmat) {
-            SlimefunItemStack chestplate = new SlimefunItemStack("MOCK_HAZMAT_SUIT_" + String.valueOf(radioactiveFire).toUpperCase(Locale.ROOT), Material.LEATHER_CHESTPLATE, "&4Hazmat Prototype");
-            MockHazmatSuit armor = new MockHazmatSuit(itemGroup, chestplate);
-            armor.register(plugin);
-
-            player.getInventory().setChestplate(chestplate.clone());
-        }
-
-        ArmorTask task = new ArmorTask(radioactiveFire);
-        task.run();
-
-        // Check if the Player is suffering from radiation
-        boolean radiation = player.getActivePotionEffects().containsAll(task.getRadiationEffects());
-        Assertions.assertEquals(!hazmat, radiation);
-        Assertions.assertEquals(!hazmat && radioactiveFire, player.getFireTicks() > 0);
-    }
-
-    /**
-     * This returns an {@link Arguments} {@link Stream} of boolean combinations.
-     * It performs a cartesian product on two boolean sets.
-     * 
-     * @return a {@link Stream} of {@link Arguments}
-     */
-    private static Stream<Arguments> cartesianBooleans() {
-        Stream<Boolean> stream = Stream.of(true, false);
-        return stream.flatMap(a -> Stream.of(true, false).map(b -> Arguments.of(a, b)));
-    }
-
 }
```

Comment:
Is there a plan to re-do the tests with all of the armor and radiation related changes?